src/main/java/com/google/cloud/spanner/pgadapter/statements/SessionStatementParser.java

```diff
@@ -279,20 +279,45 @@ static SetStatement parseSetStatement(SimpleParser parser) {
       // Ignore, this is the default.
       parser.eatKeyword("session");
     }
+    if (parser.eatKeyword("time", "zone")) {
+      builder.setName(new TableOrIndexName("TIMEZONE"));
+
+      String value = parser.parseExpression();
+      if (value == null) {
+        throw SpannerExceptionFactory.newSpannerException(
+            ErrorCode.INVALID_ARGUMENT,
+            "Invalid SET statement: " + parser.getSql() + ". Expected value.");
+      }
+      if ("default".equalsIgnoreCase(value) || "local".equalsIgnoreCase(value)) {
```

Comment:
`default` actually has a special meaning for all `SET` statements, and should not be translated here from `default` to `UTC`, but should cause the value to be set to `null`. The `SetStatement` class will then handle it from there.

The reason that it should be handled by the `SetStatement` is that the default for a connection can be different than the default of for example the server, as it can be set in the connection request.